lib/mix_unused/analyze.ex

```diff
@@ -3,21 +3,33 @@ defmodule MixUnused.Analyze do
 
   alias Mix.Task.Compiler.Diagnostic
 
+  alias MixUnused.Config
   alias MixUnused.Exports
   alias MixUnused.Tracer
 
+  @type analyzer :: module() | {module(), any()}
+
   @callback message() :: String.t()
-  @callback analyze(Tracer.data(), Exports.t()) :: Exports.t()
+  @callback analyze(Tracer.data(), Exports.t(), any()) :: Exports.t()
```

Comment:
```suggestion
  @callback analyze(Tracer.data(), Exports.t(), keyword()) :: Exports.t()
```

Reply:
For UnreachableAnalyzer, the config is a struct. So we eventually could declare the type as a `map()`, not a `keyword()`.

Reply:
I believe it is changed to struct later. So there could be another type `options/0` that would be defined as `map() | keyword()` or something like that. Or we can settle on one option only.